pkgs/tools/system/mcron/install-vixie-programs.patch

```diff
@@ -14,7 +14,7 @@ non-root without creating /var/run, etc.
         rm -f $(fpp)crontab$(EXEEXT) > /dev/null 2>&1; \
         $(INSTALL) --mode='u=rwxs,og=rx' mcron$(EXEEXT) $(fpp)crontab$(EXEEXT); \
 -       $(INSTALL) -d --mode='u=rwx' $(DESTDIR)/var/cron; \
--       $(INSTALL) -d --mode='u=rwx,og=rx' $(DESTDIR)/var/run; \
+-       $(INSTALL) -d --mode='u=rwx,og=rx' $(DESTDIR)/run; \
```

Comment:
it is a ```.patch``` file and the lines to be removed on the patch apply.
changing them means the patch apply will fail